.ci.yaml

```diff
@@ -241,6 +241,19 @@ targets:
     timeout: 120
     properties:
       config_name: linux_clang_tidy
+    runIf:
+      - DEPS
+      - .ci.yaml
+      - tools/clang_tidy/**
+      - ci/builders/**
+      - ci/lint.sh
+      - "**.h"
+      - "**.c"
+      - "**.cc"
+      - "**.fbs"
+      - "**.frag"
+      - "**.vert"
+      - "**.py"
```

Comment:
I filed https://github.com/flutter/flutter/issues/132657 to address the need for "**.py" here.